src/target/riscv/riscv-013.c

```diff
@@ -68,6 +67,13 @@ static int read_memory(struct target *target, target_addr_t address,
 static int write_memory(struct target *target, target_addr_t address,
 		uint32_t size, uint32_t count, const uint8_t *buffer);
 
+typedef enum {
+	HALTGROUP,
+	RESUMEGROUP
```

Comment:
Detail - if you'd like:

```suggestion
	HALT_GROUP,
	RESUME_GROUP
```